src/list.c

```diff
@@ -81,6 +67,22 @@ void list_add_unique(struct list_main *list, char *data)
 	list_add(list, data);
 }
 
+void list_add_multi(struct list_main *list, char *data)
+{
+	char *comma;
+
+	do {
+		if ((comma = strchr(data, ',')))
+			*comma = 0;
+
+		list_add_unique(list, data);
+
+		data = comma + 1;
```

Comment:
Just noticed a bug coming from core (added to my notes): we might be adding 1 to `NULL` here. We're not using the resulting pointer, but it's UB nevertheless. We should fix this separately.

Reply:
It might be UB but once it is, that value won't be used. Is that still a risk?

Reply:
I think formally this is still a risk. In practice, probably not.

Reply:
We should of course fix it anyway. Anyway I'm glad I'm not the only one producing bugs, just a magnitude or two more of them 🙈